DeploymentExperiences.md

```diff
@@ -0,0 +1,13 @@
+### Experience Deploying First Django App
+
+
```

Comment:
curious how you feel after getting it deployed.  Think you would be a good person to discuss with Aaron some feedback to improve.  The test on another topic completely is not what we are going for I don't think.